### DISCUSSION TITLE
import data from firebase

### DISCUSSION THREAD

**Discussion by expit-by** [appy-one/acebase-server#52] [Q&A]
tell me how to transfer data from firebase to acebase?

**Comment by appy-one** [comment]
Hi @expit-by

That depends on the size of your data and its structure. Exporting data from your Firebase db to a local json file would be the first step. If your db is not big (let's say <10MB), simply store it in your AceBase db using `db.root.set(importedData)`. Otherwise, your might want to loop through your data manually to add items 1 by 1 or in batches. Also - if you want to use AceBase's generated id's for items in your collectiosn instead of the ones generated by Firebase, you should add them 1 by 1 using `collectionRef.push(item)`.

Let me know if you have more questions.
